.gitlab-ci.yml

```diff
@@ -38,34 +38,13 @@ RPM:
     - schutzbot/save_journal.sh
   parallel:
     matrix:
-      - RUNNER:
-          - aws/fedora-33-x86_64
-          - aws/fedora-33-aarch64
-          - aws/fedora-34-x86_64
-          - aws/fedora-34-aarch64
-          - aws/rhel-8-x86_64
-          - aws/rhel-8-aarch64
-          - aws/centos-stream-8-x86_64
-          - aws/centos-stream-8-aarch64
       - RUNNER:
           - aws/rhel-8.5-x86_64
           - aws/rhel-8.5-aarch64
           - aws/rhel-9.0-beta-nightly-x86_64
           - aws/rhel-9.0-beta-nightly-aarch64
```

Comment:
Do we want rhel-9 runners here? I guess just 8.5 are enough, right? Same with base tests section.

Reply:
AFAIK at least some changes should go into 9.0 Beta, so this is actually beneficial to test on el9 as well...